mm2src/coins/eth/wallet_connect.rs

```diff
@@ -131,10 +135,8 @@ impl WalletConnectOps for EthCoin {
         };
         let tx_hash = tx_hash.strip_prefix("0x").unwrap_or(&tx_hash);
         let maybe_signed_tx = {
-            // Wait for 10 seconds for the transaction to appear on the RPC node.
-            let wait_rpc_timeout = 10_000;
             let check_every = 1.;
```

Comment:
this is the check_every that i was referring to. it's `1s` while the timeout is `10_000ms`, which is confusing.

Reply:
Will update to use secs instead